extensions/resource-deployment/src/services/toolsService.ts

```diff
@@ -15,7 +15,7 @@ export interface IToolsService {
 
 export class ToolsService implements IToolsService {
 	constructor() {
-		this.SupportedTools = [new DockerTool(), new AzCliTool(), new MSSQLCtlTool(), new KubeCtlTool()];
+		this.SupportedTools = [new DockerTool(), new AzCliTool(), new AzDataTool(), new KubeCtlTool()];
 	}
 
 	private SupportedTools: ITool[];
```

Comment:
Minor (know it's not in this PR): Why is this TitleCased? Shouldn't it be `supportedTools`?